cypress/integration/page.spec.js

```diff
@@ -236,7 +265,7 @@
         cy.get('@posts').should('not.be.called');
       });
 
-      it('should not request comments from API', () => {
+      it.skip('should not request comments from API', () => {
```

Comment:
This test is skipped. Ensure that it is intentionally skipped and consider enabling it if it is relevant to the test suite.